styles.css

```diff
@@ -603,3 +628,277 @@ textarea {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+    margin-right: 20px;
+    padding: 24px;
+  }
+
+  .hello h2 {
+    font-size: 48px;
+  }
+
+  .connect {
+    margin-left: 18%;
+  }
+
+  .socialMedia {
+    margin-right: 25%;
+  }
+
+  .tonic {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    background-size: cover;
+    background-repeat: no-repeat;
+    border: 1px solid #dfe1e6;
+    font-family: 'poppins', sans-serif;
+    padding: 16px;
+    border-radius: 16px;
+    width: 100%;
+    height: 100%;
+  }
+
+  .pic-1 {
+    display: none;
+  }
+
+  .pic-2 {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    height: 100%;
+  }
+
+  .tonicContent,
+  .tonic2Content {
+    margin-left: 5%;
+    margin-bottom: 4%;
+    width: 60%;
+  }
+
+  .photo1 {
+    display: none;
+  }
+
+  .photo2 {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    height: 100%;
+  }
+
+  .stories {
+    width: 95%;
+    margin: 0.5rem auto;
+    display: flex;
+    flex-direction: row-reverse;
+    align-items: center;
+    background-size: cover;
+    background-repeat: no-repeat;
+    border: 1px solid #dfe1e6;
+    font-family: 'poppins', sans-serif;
+    padding: 16px;
+    border-radius: 16px;
+  }
+
+  .tonicList1 {
+    display: none;
+  }
+
+  .tonicListR {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: #7a869a;
+    padding-bottom: 10px;
+    display: block;
+  }
+
+  .tonicListR p {
+    color: #7a869a;
+    font-size: 13px;
+  }
+
+  .tonicListR li {
+    display: inline;
+  }
+
+  .tonicListQ {
+    display: none;
+  }
+
+  .tonicDescription1 {
+    display: none;
+  }
+
+  .tonicDescriptionQ p {
+    font-size: 15px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .tonic2 {
+    width: 95%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    margin: 0.5rem auto;
+    background-size: cover;
+    background-repeat: no-repeat;
+    border: 1px solid #dfe1e6;
+    font-family: 'poppins', sans-serif;
+    padding: 16px;
+    border-radius: 16px;
+  }
+
+  .title1 {
+    display: none;
+  }
+
+  .title2 {
+    display: block;
+    font-family: 'Poppins', sans-serif;
+    color: #091e42;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 52px;
+  }
+
+  .stories2 {
+    width: 95%;
+    margin: 0.5rem auto;
+    display: flex;
+    flex-direction: row-reverse;
+    align-items: center;
+    background-size: cover;
+    background-repeat: no-repeat;
+    border: 1px solid #dfe1e6;
+    font-family: 'poppins', sans-serif;
+    padding: 16px;
+    border-radius: 16px;
+  }
+
+  #about {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    align-content: center;
+    margin: auto;
+    background: #fff;
+    height: 100%;
+  }
```

Comment:
Your about section has some sizing, and aligning issues. Kindly make sure to fix it.

![image](https://user-images.githubusercontent.com/34779928/194649725-6c9231e4-32d9-4a54-b141-d2aae0cdabec.png)